node/src/components/block_synchronizer.rs

```diff
@@ -387,8 +387,8 @@ impl BlockSynchronizer {
 
         match &mut self.forward {
             Some(builder) if builder.block_hash() == *block_hash => {
-                if let Some((finalized_block, deploys)) = result {
-                    builder.register_made_finalized_block(finalized_block, deploys);
+                if let Some(executable_block) = result {
+                    builder.register_made_finalized_block(executable_block);
```

Comment:
Should the function be renamed to `register_made_executable_block`?

Reply:
Fixed